druid-shell/examples/perftest.rs

```diff
@@ -118,7 +118,16 @@ fn main() {
     builder.set_handler(Box::new(perf_test));
     builder.set_title("Performance tester");
 
-    let window = builder.build().unwrap();
-    window.show();
+    // TODO(x11/architecture): super hacky way to connect the XWindow to the RunLoop. Better way to do it?
+    cfg_if::cfg_if! {
+        if #[cfg(all(target_os = "linux", feature = "use_x11"))] {
```

Comment:
I'm confused that this is still here, I thought I removed it?? In any case we want all backends to have the same public api. (Other backends currently use some global state to handle things like this; it isn't perfect but if we're going to rethink that we should do it all at once.)

Reply:
Fixed in my branch :)